test/lib/util.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('insist');
+
+function assertSecurityHeaders(res) {
+  var expect = {
+    'strict-transport-security': 'max-age=15552000; includeSubDomains',
+    'x-content-type-options': 'nosniff',
+    'x-xss-protection': '1; mode=block',
+    'x-frame-options': 'DENY',
+    'cache-control': 'private, no-cache, no-store, must-revalidate'
```

Comment:
Does it make sense to hook the cache-control value here up to config so that if config changes, the test doesn't need to be modified?